config/rethinkdb_logstash.conf

```diff
@@ -11,12 +11,17 @@ input {
 
 filter {
   mutate {
-    add_field => {"type" => "volunteer"}
+    add_field => {"type" => "volunteer"} 
     rename => {
       "new_val" => "[doc]"
     }
+    add_field => {"suggest" => "%{[doc][suggest]}"} 
```

Comment:
To pole `suggest` raczej nie będzie się znajdowało w bazie danych? Jaki był tutaj Twój zamysł?